.github/ISSUE_TEMPLATE/--bug-report.yml

```diff
@@ -0,0 +1,46 @@
+name: "🐛Bug report"
+description: "Create a bug report about GDevelop or the game engine"
+body:
+  - type: textarea
+    attributes:
+      label: Describe the bug
+      description: |
+        A clear and concise description of what the bug is.
+        Please double check that the bug is not already reported in the issues list.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Steps to reproduce
+      description: |
+        * Please include a link to a game if possible!
+        * If applicable, add screenshots to help explain your problem.
+      placeholder: |
+        1. Go to '...'
+        2. Click on '...'
+        3. Scroll down to '...'
+        4. See error
+    validations:
+      required: true
+  - type: dropdown
+    attributes:
+      label: GDevelop version
+      description: Which version of GDevelop are you used?
+      multiple: true
+      options:
+        - Desktop app
+        - Web app
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Info
+      value: |
+        OS: 
+        OS Version: 
+        Browser(For Web app): 
+        Device(For Smartphone): 
+  - type: textarea
```

Comment:
can we add screenshots in a textarea field?
If not, maybe we should add a new field especially for screenshots

Reply:
We can add screenshots or files to the textarea field! (same to comment textarea field)

Reply:
![image](https://user-images.githubusercontent.com/88178296/233582615-85854028-8f4e-4c8a-a03b-6f50bb66c407.png)